airbyte-integrations/connectors/source-facebook-pages/unit_tests/unit_test.py

```diff
@@ -16,6 +21,49 @@ def json(self):
         return json.loads(self.value)
 
 
+@pytest.fixture(name="config")
+def config_fixture():
+    config = {
+        "access_token": "TOKEN",
+        "page_id": "105762628872670",
+    }
+
+    return config
+
+
+def test_check_connection_ok(config):
+    with requests_mock.Mocker() as m:
+        m.get(f"https://graph.facebook.com/{config['page_id']}", json={"access_token": "PAGE_TOKEN"})
+        m.get(f"https://graph.facebook.com/v14.0/{config['page_id']}?access_token=PAGE_TOKEN", json={})
+    
+        ok, error_msg = SourceFacebookPages().check_connection(logger, config=config)
+
+        assert ok
+        assert not error_msg
+
+
+def test_check_connection_empty_config():
+    ok, error_msg = SourceFacebookPages().check_connection(logger, config={})
+
+    assert not ok
+    assert error_msg
```

Comment:
Consider asserting concrete error messages to distinguish between different error scenarios.

Same applies to 

```
test_check_connection_invalid_config(config)
test_check_connection_exception(config)
```

Reply:
Good deal. Updated in [96d63b8](https://github.com/airbytehq/airbyte/pull/14660/commits/96d63b8fbf8ae93b540b7d6a158f32be7795c3ff).